src/main/scala/io/iohk/ethereum/jsonrpc/TransactionReceiptResponse.scala

```diff
@@ -0,0 +1,26 @@
+package io.iohk.ethereum.jsonrpc
+
+import akka.util.ByteString
+import io.iohk.ethereum.domain.Address
+
+case class TransactionReceiptResponse(
+  transactionHash: ByteString,
+  transactionIndex: BigInt,
+  blockNumber: BigInt,
+  blockHash: ByteString,
+  cumulativeGasUsed: BigInt,
+  gasUsed: BigInt,
+  contractAddress: Option[Address],
+  logs: Seq[TxLog]
+)
+
+//TODO extract to top level because it is used in FilterManager.scala
```

Comment:
this should be addressed after https://github.com/input-output-hk/etc-client/pull/218 is merged